luigi/scheduler.py

```diff
@@ -224,6 +224,9 @@ def prune(self, config):
         if self.last_active + config.worker_disconnect_delay < time.time():
             return True
 
+    def assistant(self):
```

Comment:
It seems reasonable here, as that gives us the option to make assistant a member variable in the future.